cpp/arcticdb/processing/test/test_clause.cpp

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
```

Comment:
I think the inclusion of a completely different testing framework (even on that is contained in an existing one) is somethign we should maybe talk about before code review. Does the syntactic sugar for one row justify the additional project-wide dependency?

Reply:
Not sure why Google put the matcher functions in a gmock, but those two projects have been merged into one repo at least. I can narrow the include to `gmock-matchers.h` if it helps?